### DISCUSSION TITLE
OpenGlassGUI (GUI for OpenGlass) - Testing needed

### DISCUSSION THREAD

**Discussion by zxrohex** [ALTaleX531/OpenGlass#77] [Show and tell]
Yo,

I made a GUI for OpenGlass, it's very unfinished and still pretty rough code-wise, but it (seems) to work (for me lol), which is why I am looking for some people to test it. Also I'm looking for any kind of feedback like ideas, contributions or whatever. 

It's written in C# (.NET 9) and the GUI is made using WinForms.

You need to install OpenGlass into C:\OpenGlass\ and install it using the install.bat included before starting OpenGlassGUI! This will be changed in the future, including an auto-setup added in the future (depending on the owner/license I may include it or something but IDK yet). Framework is bundled.

Tested on Windows 11 Dev Insider Preview. Should work on Win 11 Stable. No support for Win 10 yet.

[https://github.com/zxrohex/OpenGlassGUI](https://github.com/zxrohex/OpenGlassGUI)

**Comment by Milicent-Heeler** [comment]
I can test it out for you!

**Reply by zxrohex** [reply]
Awesome! Thank you. You can download an testing release in the GitHub repo.

**Comment by Milicent-Heeler** [comment]
But, does it support openglass Dcomp? If not then I can't.

**Reply by zxrohex** [reply]
I built it using the master branch as the target and for reference, so yes it does, which is also the only supported for now.

**Reply by Milicent-Heeler** [reply]
> I built it using the master branch as the target and for reference, so yes it does, which is also the only supported for now.

Ok, Is it Release 0.0.0.0-testing Pre ALpha?

**Reply by Milicent-Heeler** [reply]
> I built it using the master branch as the target and for reference, so yes it does, which is also the only supported for now.

Please reply back.

**Comment by Milicent-Heeler** [comment]
I tried out 1 of the settings, hit apply but the setting didn't apply and because of that, I logged out and back in but then it made openglass become a solid colour.

**Reply by Milicent-Heeler** [reply]
> I tried out 1 of the settings, hit apply but the setting didn't apply and because of that, I logged out and back in but then it made openglass become a solid colour.

Also I have Win11 23H2 Home Single Language.

**Comment by ALTaleX531** [comment]
Thank you very much for the effort you put into creating your own GUI before the official one was available. Your contribution helped the community a lot and showed great initiative. We now have the official GUI released and ready to use, but I truly appreciate the work you did earlier to fill the gap.
